indico/MaKaC/PDFinterface/conference.py

```diff
@@ -1636,7 +1639,9 @@ def getBody(self, story=None, indexedFlowable={}, level=1 ):
             for item in self._display:
                 if item.input_type == 'accommodation':
                     if item.id in data:
-                        lp.append(Paragraph(data[item.id].friendly_data.get('choice').encode('utf-8'), text_format))
+                        choice = data[item.id].friendly_data.get('choice')
```

Comment:
You could also do it like this:

    choice = data[item.id].friendly_data.get('choice', '').encode('utf-8')